api/tests/connectors/cgr/api_cgr_test.py

```diff
@@ -0,0 +1,117 @@
+                                "bAvecDuo": True,
+                                "bICE": True,
+                                "Relief": "2D",
+                                "Version": "VF",
+                                "bAVP": False,
+                                "PrixUnitaire": 11,
+                            },
+                        ],
+                    },
+                ],
+            },
+        }
+        response = f"""
+        <?xml version="1.0" encoding="UTF-8"?>
+        <SOAP-ENV:Envelope
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/">
+            <SOAP-ENV:Header/>
+            <SOAP-ENV:Body>
+                <ns1:GetSeancesPassCultureResponse xmlns:ns1="urn:GestionCinemaWS">
+                    <GetSeancesPassCultureResult>
+                        {json.dumps(json_response)}
+                    </GetSeancesPassCultureResult>
+                </ns1:GetSeancesPassCultureResponse>
+            </SOAP-ENV:Body>
+        </SOAP-ENV:Envelope>
+        """.strip()
+        requests_mock.post("http://example.com/web_service", text=response)
+
+        result = get_seances_pass_culture()
+
+        assert result.CodeErreur == 0
+        assert result.IntituleErreur == ""
+        assert result.ObjetRetour.NumCine == 999
+        assert isinstance(result.ObjetRetour.Films, list)
+
+    @override_settings(
+        CGR_API_URL="http://example.com/web_service", CGR_API_USER="pass_user", CGR_API_PASSWORD="password"
+    )
+    def test_should_raise_if_error(self, requests_mock):
+        requests_mock.get("http://example.com/web_service?wsdl", text=soap_definitions.WEB_SERVICE_DEFINITION)
+        json_response = {"CodeErreur": -1, "IntituleErreur": "Expectation failed", "ObjetRetour": None}
+        response = f"""
+        <?xml version="1.0" encoding="UTF-8"?>
+        <SOAP-ENV:Envelope
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/">
+            <SOAP-ENV:Header/>
+            <SOAP-ENV:Body>
+                <ns1:GetSeancesPassCultureResponse xmlns:ns1="urn:GestionCinemaWS">
+                    <GetSeancesPassCultureResult>
+                        {json.dumps(json_response)}
+                    </GetSeancesPassCultureResult>
+                </ns1:GetSeancesPassCultureResponse>
+            </SOAP-ENV:Body>
+        </SOAP-ENV:Envelope>
+        """.strip()
+        requests_mock.post("http://example.com/web_service", text=response)
+
+        with pytest.raises(cgr_exceptions.CGRAPIException) as exc:
+            get_seances_pass_culture()
+
+        assert isinstance(exc.value, cgr_exceptions.CGRAPIException)
+        assert "Error on CGR API on GetSeancesPassCulture : Expectation failed" == str(exc.value)
```

Comment:
```suggestion
        assert str(exc.value) == "Error on CGR API on GetSeancesPassCulture : Expectation failed"
```
J'aurais inversé les termes, mais c'est par définition une égalité :)